Api/Data/Fl32RuleSentInterface.php

```diff
@@ -0,0 +1,67 @@
+<?php
+declare(strict_types=1);
+
+namespace Flancer32\Csp\Api\Data;
+
+interface Fl32RuleSentInterface extends \Magento\Framework\Api\ExtensibleDataInterface
```

Comment:
Duplicating the top level namespace in the class name (`Fl32`) is not necessary, also wasn't done by the original author. I guess the reason for this naming was that this was originally built as an add-on module

Reply:
The same goes for constant names, function names etc., `FL32` should be removed. For database tables however, the prefix makes sense

Reply:
These are generated classes, however the change was applied

Reply:
Apparently the function name is used for some magic, which was also the reason why the _ as a separator between number and string did not work as expected. Therefor I had to reintroduce the fl32 substring inside the dataobject functions. Otherwise they would not refer to the table columns correctly.

Reply:
OK for method names I understand. I wouldn't add the prefix to column names in the first place but if the original module does this too (does it?), it makes sense for consistency.

Reply:
It is not consistent with the original tables. Would you then change this?

Reply:
yes please